io/gcs.go

```diff
@@ -46,6 +49,9 @@ func ParseGCSConfig(props map[string]string) *gcsblob.Options {
 	if path := props[GCSKeyPath]; path != "" {
 		o = append(o, option.WithCredentialsFile(path))
 	}
+	if _, ok := props[GCSUseJsonAPI]; ok {
+		o = append(o, storage.WithJSONReads())
+	}
```

Comment:
is this the only reason why we're pulling in `cloud.google.com/go/storage`? I think that's okay for now, but we may want to see if we potentially contribute to `gocloud.dev/gcp` instead of pulling an entire other library in